python/testEditorInterface.py

```diff
@@ -52,7 +58,7 @@ def main():
                 choiceData = choiceDataType()
 
         elif mode == "choiceList-tst":
-            choiceData = processTstLine(enviroName, inputLine)
+            choiceData = processTstLine(enviroName, inputLine, additionalParams)
```

Comment:
I don't even understand what is being done here yet but adding a new parameter cannot be the best way to do this

Reply:
I thought this would be the best way since 
 - I need the `unit` to get all the functions for the TEST.SUBPROGRAM with `getFunctions()`.
 - The changes are not really huge 
 
 I brainstormed some ideas in the notes here: 
 https://github.com/vectorgrp/vector-vscode-vcast/pull/193
 
 I basically implemented idea **1)** because I thought it s the best of those. If you want me to, I could take again a look at it.